challenges/objects-arrays.js

```diff
@@ -6,24 +6,63 @@
   object name, diet, weight, length, period
 */
 
+class Dinosaur {
+  constructor(attribs) {
+    this.name = attribs.name;
+    this.diet = attribs.diet;
+    this.weight = attribs.diet;
+    this.length = attribs.diet;
```

Comment:
It looks like you made a mistake here. Let me know if you can't see what it is. Hint: the diet, weight, and length in your console logs are ALL logging the dinosaur's diet.